### PR TITLE
Give a little breath noise whenever you put on a gas mask.

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
When you put it on, you get a breathy noise. It is fluff. Does not happen afterward.

## Why It's Good For The Game
Fluff good. Sounds good.

## Changelog
:cl:
soundadd: added a breathy noise whenever you equip a gas mask.
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
Add in a slight cooldown. I don't wanna be vader-spammed on the Alamo.

**Comment by InterroLouis** [comment]
What if I do in fact, want to be vader-spammed on the Alamo.

**Comment by XSlayer300** [comment]
It can be annoying at times tho, wish we have an option to switch it off.

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -15,6 +15,17 @@
 	siemens_coefficient = 0.9
 	var/gas_filter_strength = 1			//For gas mask filters
 	var/list/filtered_gases = list(/datum/reagent/toxin/phoron, "sleeping_agent", "carbon_dioxide")
+	var/breathy = 1		///Whether or not this plays a sound on equip
```
Comment (BraveMole) [inline code/modules/clothing/masks/gasmask.dm:18]: TRUE / FALSE
also, you know that this is not how we autodoc

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by Sebbe9123** [comment]
How long is the sound file? Is it something like three breaths or is it a few minutes of breathing?

**Comment by lbnesquik** [comment]
> 
> 
> How long is the sound file? Is it something like three breaths or is it a few minutes of breathing?

You can open the sounfiles for a listen. It is two breaths

File: code/modules/clothing/masks/gasmask.dm (hunk 1 above)
Comment (lbnesquik) [reply code/modules/clothing/masks/gasmask.dm:18]: Done.

**Comment by SplinterGP** [comment]
i'd like if it kept playing throgh the round randomly, but i get its hard to do properly.

**Review by TiviPlus — APPROVED** [review]

**Comment by Al-1ce** [comment]
This sounded cool but in play it isn't that great. I'd advise a revert.

**Pull request merged**